contracts/.env.example

```diff
@@ -9,3 +9,6 @@ XDAI_JSONRPC_HTTP_URL=https://rpc.xdaichain.com
 
 WALLET_MNEMONIC=
 WALLET_PRIVATE_KEY=
+
+# Token address for funding round .setToken
+TOKEN_ADDRESS=
```

Comment:
```suggestion
NATIVE_TOKEN_ADDRESS=
```

Reply:
Just to make it clear this is referring to the `nativeToken` of the `FundingRoundFactory` contract.